composer.json

```diff
@@ -26,6 +26,7 @@
         "symfony/framework-bundle": "~2.3"
     },
     "require-dev": {
+        "friendsofsymfony/http-cache": "~1.2@dev",
```

Comment:
don't put the same requirement in both require and require-dev (Composer warns about it). Put the dev flag directly in require (it will only affect the requirement when being in the root package anyway)